tests/main/snap-discard-ns/mount-py2.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python2
+"""
+Pure-python limited-use wrapper around the mount library function.
+"""
+from __future__ import print_function, absolute_import, unicode_literals
+
+from argparse import ArgumentParser
+from ctypes import CDLL, c_char_p, c_long, get_errno
+from ctypes.util import find_library
+from os import strerror
+from sys import stderr
+
+__all__ = ('mount', )
+
+
+class MountOpts(object):
+    """MountOpts contain various flags for the mount system call."""
+
+    Bind = 4096
+
+
+def mount(source, target, fstype, flags=0, data=""):
+    # type: (unicode, unicode, unicode, int, unicode) -> None
+    """mount is a thin wrapper around the mount library function."""
+    libc_name = find_library(b"c")
+    if libc_name is None:
+        raise Exception("cannot find the C library")
+    libc = CDLL(libc_name, use_errno=True)
+    retval = libc.mount(
+            c_char_p(source.encode("UTF-8")),
```

Comment:
`utf-8` is the default, so you may drop that.

Reply:
UTF-8 is not the default.

Reply:
I think it is in Py3. In Py2 it was derived from your locale or sth. OTOH, I'm not sure we should be forcing utf-8 here which may not be the same as input encoding so the input path provided to the script will be a different byte sequence from one encoded to utf-8.